build/constants.yaml

```diff
@@ -189,19 +189,19 @@
       directory: src/startupscriptgenerator/src/common/consts
 - name: node-versions
   constants:
-    node-runtime-base-tag: 20210902.2
-    yarn-version: 1.22.10
+    node-runtime-base-tag: 20211208.1
+    yarn-version: 1.22.15
     yarn-minor-version: 1.17
     yarn-major-version: 1
     node6-version: 6.17.1
     node8-version: 8.17.0
     node10-version: 10.23.0
     node12-version: 12.22.6
     node14-version: 14.17.6
-    node16-version: 16.8.0
+    node16-version: 16.13.1
     node-app-insights-sdk-version: 1.8.7
     pm2-version: 4.5.6
-    npm-version: 6.14.15
+    npm-version: 8.1.2
```

Comment:
where is this version used? are we sure it is compatible with the node versions we use it with?

Reply:
@gpcastro npm 8.1.2 is used during the builds:
![image](https://user-images.githubusercontent.com/86327553/145658131-e89b58f0-54ba-4d91-9dfa-af844a29489e.png)
This is built successfully and will run oryx-nightly to make sure E2E tests are passing successfully